_dev/js/back/src/App.vue

```diff
@@ -0,0 +1,257 @@
+<!--**
+ * 2007-2020 PrestaShop and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Academic Free License 3.0 (AFL-3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/AFL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2020 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/AFL-3.0 Academic Free License 3.0 (AFL-3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ *-->
+<template>
+  <div id="app">
+    <Menu>
+      <MenuItem route="/authentication">
+        {{ $t("menu.authentication") }}
+      </MenuItem>
+      <template
+        v-if="onboardingPaypalIsCompleted && onboardingFirebaseIsCompleted"
+      >
+        <MenuItem route="/customize">
+          {{ $t("menu.customizeCheckout") }}
+        </MenuItem>
+        <MenuItem route="/activity">
+          {{ $t("menu.manageActivity") }}
+        </MenuItem>
+        <MenuItem route="/advanced">
+          {{ $t("menu.advancedSettings") }}
+        </MenuItem>
+      </template>
+      <MenuItem route="/help">
+        {{ $t("menu.help") }}
+      </MenuItem>
+    </Menu>
+
+    <div class="pt-5" />
+    <div class="pt-3" />
+
+    <div class="container" v-if="isShopContext">
+      <RoundingBanner />
+    </div>
+
+    <div class="container" v-if="isTestMode">
+      <b-alert variant="warning" show>
+        <p>{{ $t("general.testModeOn") }}</p>
+      </b-alert>
+    </div>
+
+    <div class="container" v-if="!isShopContext">
+      <b-alert variant="warning" show>
+        <h2>{{ $t("general.multiShop.title") }}</h2>
+        <p>{{ $t("general.multiShop.subtitle") }}</p>
+        <p>{{ $t("general.multiShop.chooseOne") }}</p>
+        <b-list-group
+          v-for="group in shopsTree"
+          :key="group.id"
+          class="mt-3 mb-3 col-4"
+        >
+          <p class="text-muted">
+            {{ $t("general.multiShop.group") }} {{ group.name }}
+          </p>
+          <b-list-group-item
+            v-for="shop in group.shops"
+            :key="shop.id"
+            :href="shop.url"
+          >
+            {{ $t("general.multiShop.configure") }} <b>{{ shop.name }}</b>
+          </b-list-group-item>
+        </b-list-group>
+        <p>{{ $t("general.multiShop.tips") }}</p>
+      </b-alert>
+    </div>
+
+    <router-view v-if="isShopContext" />
+  </div>
+</template>
+
+<script>
+import Menu from "@/components/menu/menu";
```

Comment:
The `/` after the `@` looks so weird, what about adding it to the webpack path instead of in every files ?

Reply:
I don't know, I try but I have an error : `Module is not installed` 🤷‍♂️

Reply:
On webpack?